frontend/src/components/dashboard/aliases/AliasList.tsx

```diff
@@ -36,9 +36,25 @@ export const AliasList = (props: Props) => {
   const [stringFilterVisible, setStringFilterVisible] = useState(false);
   const [categoryFilters, setCategoryFilters] = useState<SelectedFilters>({});
   const [localLabels, storeLocalLabel] = useLocalLabels();
-  // Whenever a new alias is created, this value tracks the aliases that existed
-  // before that. That allows us to expand newly-created aliases by default.
-  const [existingAliases, setExistingAliases] = useState(props.aliases);
+  const [openAlias, setOpenAlias] = useState<AliasData | undefined>(undefined);
+  const [existingAliases, setExistingAliases] = useState<AliasData[]>(
+    props.aliases
+  );
+
+  useEffect(() => {
+    if (props.aliases.length == 0) {
```

Comment:
Suggestion: `===` instead of `==` so that it checks that only both type and value are the same.